demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -637,22 +637,27 @@ def _collect_indicators_types_dependencies(pack_indicators_types: list, id_set:
             indicator_type_data = next(iter(indicator_type.values()))
             indicator_type_dependencies = set()
 
-            related_integrations = indicator_type_data.get('integrations', [])
-            packs_found_from_integrations = PackDependencies._search_packs_by_items_names(
-                related_integrations, id_set['integrations'], exclude_ignored_dependencies)
-
-            if packs_found_from_integrations:
-                pack_dependencies_data = PackDependencies. \
-                    _label_as_optional(packs_found_from_integrations)
-                indicator_type_dependencies.update(pack_dependencies_data)
+            #########################################################################################################
+            # Do not collect integrations implementing reputation commands to not clutter CommonTypes and other packs
+            # that have a indicator type using e.g `ip` command with all the reputation integrations.
+            #########################################################################################################
+
+            # related_integrations = indicator_type_data.get('integrations', [])
+            # packs_found_from_integrations = PackDependencies._search_packs_by_items_names(
+            #     related_integrations, id_set['integrations'], exclude_ignored_dependencies)
+            #
+            # if packs_found_from_integrations:
+            #     pack_dependencies_data = PackDependencies. \
+            #         _label_as_optional(packs_found_from_integrations)
+            #     indicator_type_dependencies.update(pack_dependencies_data)
 
             related_scripts = indicator_type_data.get('scripts', [])
             packs_found_from_scripts = PackDependencies._search_packs_by_items_names(
                 related_scripts, id_set['scripts'], exclude_ignored_dependencies)
 
             if packs_found_from_scripts:
                 pack_dependencies_data = PackDependencies. \
-                    _label_as_mandatory(packs_found_from_scripts)
+                    _label_as_optional(packs_found_from_scripts)
```

Comment:
this means that for CommonScripts will have optional dependencies to all the integrations with reputation commands, because of `IpReputation` script uses `ip` command?

Reply:
No, as we are only collecting first level dependencies.